src/marketing/newsletter-configuration.md

```diff
@@ -2,18 +2,20 @@
 title: Configuring Newsletters
 ---
 
-The first step in creating newsletters is to configure the newsletter settings for your site. You can require customers to click a confirmation link that is sent by email to confirm the subscription. This double opt-in, method requires customers to confirm twice that they want to receive your newsletter, and reduces the possibility that it might be considered to be spam.
+The first step in creating newsletters is to configure the newsletter settings for your site. You can require customers to click a confirmation link that is sent by email to confirm the subscription. This double opt-in method requires customers to confirm twice that they want to receive your newsletter, and reduces the possibility that it might be considered to be spam.
 
-## Configure Subscription Options
+## Configure the subscription options
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
 1. In the left panel, expand **Customers** and choose **Newsletter**.
 
-1. If necessary, expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Subscription Options** section. Then, do the following:
+1. If needed, [change the configuration scope]({% link configuration/scope-change.md %}) to apply the newsletter configuration changes to a specific site/store view.
 
-    ![Customers configuration - newsletter subscriptions]({% link images/images/config-customers-newsletter-subscription-options.png %}){: .zoom}
-    [_Subscription Options_]({% link configuration/customers.md -%})
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Subscription Options** section and do the following:
```

Comment:
```suggestion
1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the _Subscription Options_ section and do the following:
```

Reply:
Since they have to touch it to expand it it should be bold.